src/Mapper/BaseGroupedMapper.php

```diff
@@ -17,6 +17,9 @@
  * This class is used to simulate the Form API.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @phpstan-template T of \Sonata\AdminBundle\Builder\BuilderInterface
```

Comment:
why you use here the full class name? On this same Pr there are other cases where you dont

Reply:
I maybe made a mistake but I use the full class name when the class is never imported.
Because phpstorm report as unused the class if it's only use for a phpstan annotation (and it's confusing IMHO).